drip/admin.py

```diff
@@ -38,6 +38,18 @@ class DripAdmin(admin.ModelAdmin):
     ]
     form = DripForm
     users_fields = []
+    actions = [
+        'action_enable_drips',
+        'action_disable_drips',
+        'action_duplicate_drips',
+    ]
+
+    def get_exclude(self, request, obj=None):
```

Comment:
please add some documentation here to explain why you are excluding these fields